Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -419,13 +430,13 @@ class SettingsTableSectionHeaderFooterView: UITableViewHeaderFooterView {
                 case .Top:
                     titleLabel.snp_remakeConstraints { make in
                         make.left.equalTo(self).offset(SettingsTableSectionHeaderFooterViewUX.titleHorizontalPadding)
-                        make.right.greaterThanOrEqualTo(self).offset(-SettingsTableSectionHeaderFooterViewUX.titleHorizontalPadding)
+                        make.right.greaterThanOrEqualTo(self).offset(-SettingsTableSectionHeaderFooterViewUX.titleHorizontalPadding).priorityLow()
```

Comment:
When I remove the `priorityLow()` call I get autolayout constraint errors because we have both this constraint and the one we add during the dequeue call. Reducing the priority seems like a patch over a bigger issue.

I did some additional digging and found the real problem. For all of these calls where we make right `greaterThanOrEqualTo`, we should actually my calling `lessThanOrEqualTo` because the offset we're setting is a negative value. It looks correct because we do want the value to grow but because its a negative value its actually getting smaller. After changing these to less than and remove the constraint in the dequeue call everything works as expected :dancer: